src/components/ClusterTemplatesGettingStarted/WithClusterTemplateQuickStarts.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useAddAlertOnError } from '../../alerts/useAddAlertOnError';
+import WithQuickStarts from '../../helpers/GettingStarted/WithQuickStarts';
+import { useK8sWatchResource } from '../../hooks/k8s';
+import { QuickStart } from '@patternfly/quickstarts';
+import { useAlerts } from '../../alerts/AlertsContext';
+import { QuickStartKey, quickStartsData } from './quickStartConstants';
+import { quickStartGVK } from '../../constants';
+
+//t('Failed to load quick starts')
+const quickStartLoadErrMessage = 'Failed to load quick starts';
+
+const WithClusterTemplateQuickStarts = ({ children }: { children: React.ReactNode }) => {
+  const { addAlert } = useAlerts();
+  const [quickStarts, quickStartsLoaded, quickStartsError] = useK8sWatchResource<QuickStart[]>({
+    groupVersionKind: quickStartGVK,
+    isList: true,
+  });
+  React.useEffect(() => {
+    if (!quickStartsLoaded || quickStartsError) {
+      return;
+    }
+
+    const unfoundQuickStarts = (Object.keys(quickStartsData) as QuickStartKey[]).reduce<string[]>(
```

Comment:
maybe rename to `missingQuickStarts` ?

```suggestion
    const missingQuickStarts = (Object.keys(quickStartsData) as QuickStartKey[]).reduce<string[]>(
```